Gujarati/gujarati.md

```diff
@@ -6,3 +6,7 @@ A problem is achance for you to do your best.
 Tha best way to predict the future is to create.
 
 Gujarati language is great. Lord Krishna spoke or led to it. Mahatma Gandhi spoke and wrote in it.
+
+ગુજરાતી ભાષાનાં પરાક્રમી સફરોમાં આપનું સ્વાગત છે. અગણીત સફરો માંથી તમે કયું પસંદ કરશો? તમારી ખરી કસોટી શરુ થાય છે. તમને ખુબ શુભકામના.
```

Comment:
This line is longer than 120 characters. Could you break it up into multiple lines? That way it will be easier to read the raw files or view diffs on the command line.  No need to add blank lines in between, just press enter somewhere in the middle of the line.